CHANGELOG.rst

```diff
@@ -6,6 +6,8 @@ History
 0.13.0 (unreleased)
 ===================
 
+* Add black-oil properties to PVT Correlations case description ```PvtModelCorrelationDescription```
```

Comment:
I think this should be on the 0.14.0 realease (since it is not in the RB).

Reply:
Sorry, maybe I am wrong.
This is a change FROM rb to be merged into master. (right?)